charts/pdf-generator/values.yaml

```diff
@@ -0,0 +1,33 @@
+replicaCount: 2
+
+environment: development
+
+autoscaling:
+  enabled: true
+  replicas:
+    min: 2
+    max: 10
+  avgCpuUtilization: 70
+  behavior:
+    stabilizationWindowSeconds:
+      scaleUp: 0
+      scaleDown: 120
+
+resources:
+  requests:
+    cpu: 30m
+    memory: 128Mi
+
+image:
+  # Set "repository" name of your image for manual Helm install and upgrade.
+  repository: browserless/chrome
+  tag: latest
```

Comment:
It's recomended to not depend on latest as we have no control if there are breaking changes in latest. Should we rather depend on a pinned version and upgrade the chart when new versions are available?